### ISSUE TITLE
test contact page

### ISSUE THREAD

**Issue by Hayyanebr** [MahmoudAljabary/KMS#12]

**Comment by figaru** [comment]
hey man why all these issues opened?

**Comment by Hayyanebr** [comment]
coz I dont see anything in the project

**Comment by figaru** [comment]
yep no one has done anything yet... ive been doing designs but its all done seperatly just css and html. But the people doing the back end do not need those dseigns to work as everything as mockups have been added to the drive/research/mockups

**Comment by figaru** [comment]
Pascal (Database manager): 
•	Create database SQL
•	Insert data into the database (Posts, users, user info, login info, skills, categories, etc)
•	Export database, share with group on Facebook.
•	Add column for category colours.
•	Add table for articles. 
•	Add field for comment parents.
•	Add field for admin status. 

Deadline: Sunday May 1st

If another member wished to make changes to the database, they must message Pascal.


Daniel (Design manager):
•	Design profile page
•	Design user form
•	Design notifications
•	Design search page
•	Design create post page
•	Design check post pop up

Deadline: Wednesday 4th May

If another member wished to make changes to the design, they must message Daniel.


John (Team Leader):
•	Create Pack & Go logo
•	Create thesis outline
•	Create user form sketch
•	Create sketches for iOS & Android apps

Deadline: Wednesday 4th May


Mahmoud (Posting guru):
•	Create public GitHub for group, invite team to it, share link on Facebook
•	Create functional posting (Post title, description, category, tags, user)
•	Editing posts

Deadline: Tuesday 10th May


Hafiz (User guru):
•	Create working user form
•	Create working log in
•	Give admin access
•	Send Daniel an example of a Laravel project

Deadline: Tuesday 10th May
Patrick (Search guru):
•	Functional search
•	Functional search tools
•	Functional search page 
•	Finish sequence diagram

Deadline: Tuesday 10th May


Hayyan (Comment & Like guru):
•	Create functional comments
•	Create functional replies	
•	Create functioning replies

**Comment by Hayyanebr** [comment]
yes what you just sent   .. should be on github so we all see and know what is going on .....

**Comment by Hayyanebr** [comment]
we can not start anything if we dont know the Database

**Comment by Hayyanebr** [comment]
if we all work in github we all know the process .... just pick an issue create a branch with this issue name and start working on it
